ground_truth_labeling_jobs/multi_modal_parallel_sagemaker_labeling_workflows_with_step_functions/sagemaker_ground_truth_workflows.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>Please reword: "Now we'll send our created manifest through a custom SMGT pipeline, orchestrated by batch jobs." to "Now we'll send the input manifest files that we just created through a custom Ground Truth pipeline, orchestrated by batch jobs.</p><p> </p><p>Note: SMGT is not a valid short-name for ground truth. Please us "Ground Truth" here.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>